src/MongoDB/ClientEncryption.c

```diff
@@ -120,7 +120,7 @@ ZEND_BEGIN_ARG_INFO_EX(ai_ClientEncryption_encrypt, 0, 0, 1)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO_EX(ai_ClientEncryption_decrypt, 0, 0, 1)
-	ZEND_ARG_OBJ_INFO(0, keyVaultClient, MongoDB\\BSON\\BinaryInterface, 0)
+	ZEND_ARG_OBJ_INFO(0, value, MongoDB\\BSON\\BinaryInterface, 0)
```

Comment:
Note that this will become obsolete once #1337 is merged.

Reply:
This was removed after rebasing on master.